### PR TITLE
Update staging Gitlab to the latest helm chart version

### PR DESCRIPTION
**Pull request by opadron**

### CODE REVIEW

File: images/gitlab-toolbox/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-task-runner-ee@sha256:c643675862fc5b3dac0a75329ddd12cba51fef2e50e5fbc69a5f14e1b7af0b92
+FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-toolbox-ee:v14.6.3
```
Comment (scottwittenburg) [inline images/gitlab-toolbox/Dockerfile:1]: What's the deal with this, I guess it's not called `task-runner` anymore?

File: images/gitlab-sidekiq/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-sidekiq-ee:v14.4.1
+FROM registry.gitlab.com/gitlab-org/build/cng/gitlab-sidekiq-ee:v14.6.3
 
 RUN ss='s/return false unless Feature.enabled?(:ci_yaml_limit_size.*' \
   ; ss="$ss/return false/g" \
```
Comment (scottwittenburg) [inline images/gitlab-sidekiq/Dockerfile:4]: I'm curious if these are still working, do you have any idea if these patch still do anything?

**Review by scottwittenburg — APPROVED** [review]
Looks good to me at a glance.

File: images/gitlab-toolbox/Dockerfile (hunk 1 above)
Comment (opadron) [reply images/gitlab-toolbox/Dockerfile:1]: That's right, they renamed it to `gitlab-toolbox`.

File: images/gitlab-sidekiq/Dockerfile (hunk 2 above)
Comment (opadron) [reply images/gitlab-sidekiq/Dockerfile:4]: I tested these builds locally to make sure that the patch at least applies.  Will have to test in staging to see if it works.

**Pull request merged**